src/app/command-channel.service.ts

```diff
@@ -112,6 +113,15 @@ export class CommandChannelService implements OnDestroy {
           NotificationType.WARNING, 'WebSocket Error', JSON.stringify(evt), false, null, null
         ));
 
+        // FIXME handle the case of a failed 'connect' command
```

Comment:
What's the impact and workaround if any of a failed connect command? Would it be warranted to open an issue to track this?

Reply:
It'll just be basically silently swallowed, and not reflected by the `isConnected` Observable. I need to come back around to this and think about how the whole `isReady/isConnected` is handled, because this error handling also exposes the need for an `isErrored` kind of status, but that's all going to be a bit awkward for components to actually consume. Especially because currently `isReady` encapsulates the state of the WebSocket command channel connection itself, whereas `isConnected` is about the state of the connection from container-jfr to the target, so the error state would need to differentiate between these connections.

Maybe what I should do is refactor it again (later) to be an `Observable<ConnectionState>`, and `ConnectionState` should be an enum of all the types of states the connection could be in, ie `no command channel connection | command channel connected but no target | command channel and target connected | command channel error | target connection error`. Not really sure, needs more thought. I'll file an issue for it.

Reply:
Filed: #49

Reply:
Okay, thank you!